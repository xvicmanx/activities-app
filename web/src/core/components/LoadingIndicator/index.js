import React from 'react';
import { Loader } from 'react-bulma-components';

const styles = {
  container: {
    textAlign: 'center',
    backgroundColor: '#fff',
    padding: '10px',
    borderRadius: '5px',
    width: 'fit-content',
    margin: '20px auto',
    color: '#5caaf6',
    lineHeight: '1.6',
    border: '1px solid #efefef',
  },
  loader: {
    border: '3px solid #5caaf6',
    width: '50px',
    height: '50px',
    display: 'block',
    margin: 'auto',
  },
};

const LoadingIndicator = () => (
  <div style={styles.container}>
    <Loader style={styles.loader} />
    Cargando ...
  </div>
);

export default LoadingIndicator;
