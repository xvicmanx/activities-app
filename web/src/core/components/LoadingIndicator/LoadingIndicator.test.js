import React from 'react';
import renderer from 'react-test-renderer';
import LoadingIndicator from './';

describe('LoadingIndicator', () => {
  it('renders properly', () => {
    const component = renderer.create(
      <LoadingIndicator />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});