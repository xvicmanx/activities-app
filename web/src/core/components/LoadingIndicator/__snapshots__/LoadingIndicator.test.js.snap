// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator renders properly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "border": "1px solid #efefef",
      "borderRadius": "5px",
      "color": "#5caaf6",
      "lineHeight": "1.6",
      "margin": "20px auto",
      "padding": "10px",
      "textAlign": "center",
      "width": "fit-content",
    }
  }
>
  <div
    className="loader"
    style={
      Object {
        "border": "3px solid #5caaf6",
        "display": "block",
        "height": "50px",
        "margin": "auto",
        "width": "50px",
      }
    }
  />
  Cargando ...
</div>
`;
